virtual-setup/zoom-procedures.md

```diff
@@ -190,24 +190,28 @@ You will also want to share your screen during your presentation at the end of t
 
 To share your screen, select the `Share Screen` icon in the middle of the control bar at the bottom of the Zoom window.
 
-<img src = screenshots/zoom-participant-share-screen.png>
+<img src = "screenshots/zoom-participant-share-screen.png">
 
 This will bring up a prompt that will allow you to select whether to share your whole desktop or a specific window.
 You most likely want to share the browser window with the RStudio interface only.
 
-<img src = screenshots/zoom-screen-window.png>
+<img src = "screenshots/zoom-screen-window.png">
 
 Other participants in the meeting (or just the breakout room if you are in one) should now be able to see your screen.
 
 To ask for help from the meeting host while in a breakout room, use the `Ask for Help` icon in the controls.
 
-<img src = screenshots/zoom-breakout-ask-for-help.png width = 500>
+<img src = "screenshots/zoom-breakout-ask-for-help.png" width = "750">
 
 ### Returning to the main session
 
-To return to the main session, select `Leave Breakout Room` from the control panel.
+To return to the main session, stop sharing your screen.
 
-<img src = screenshots/zoom-breakout-leave-room.png width = 500>
+<img src = "screenshots/zoom-breakout-stop-screen-sharing.png" width = "750">
+
+Then select `Leave Breakout Room` from the control panel.
+
+<img src = "screenshots/zoom-breakout-leave-room.png" width = "750">
```

Comment:
What if you moved the instructions about stopping sharing to the section above on sharing your screen (maybe before covering ask for help because I think that also won't show up when you are sharing your screen?) and then added something (probably in bold) in this section like "note you'll need to stop sharing your screen to leave the breakout room"

Reply:
Good point, will do.